logging/examples_test.go

```diff
@@ -86,6 +87,23 @@ func ExampleClient_Logger() {
 	_ = lg // TODO: use the Logger.
 }
 
+func ExampleHTTPRequest() {
```

Comment:
should there be a region tag around this?

Reply:
No need for region tags. These are godocs examples so its auto appended inline with the type HTTPRequest definition

You can preview how that might look with `godoc -http=:6060`

Reply:
It seems a bit strange to maintain samples in two places. I wonder if we could combine them. Maybe make synthtool pull in samples from the `golang-samples`?

That's definitely out of scope for now though!

Reply:
yeah good pt. I am wondering about this as well. IT would require that all languages also include compilable / testable samples in documentation. Interesting problem to solve for sure.